flopy/mf6/mfpackage.py

```diff
@@ -23,10 +23,11 @@
 from ..pakbase import PackageInterface
 from .data.mfdatautil import MFComment
 from ..utils.check import mf6check
+from .utils.output_util import MF6Output
 from ..version import __version__
 
 
-class MFBlockHeader:
+class MFBlockHeader(object):
```

Comment:
Aren't we eliminating object from our class definitions?  Or is this one required for some reason?

Reply:
This was added by black during linting. Is there a way to disable that?